src/Fields/broadcast.jl

```diff
@@ -301,6 +301,8 @@ Base.Broadcast.broadcasted(
     ::Val{n},
 ) where {n} = Base.Broadcast.broadcasted(x -> Base.literal_pow(^, x, Val(n)), f)
 
+Base.Broadcast.broadcasted(::Type{T}, x::Number) where {T <: Number} = x
```

Comment:
```suggestion
Base.Broadcast.broadcasted(::Type{T}, x::Number) where {T <: Number} = T(x)
```
I guess we should actually do this instead